js/islandora_book_reader.js

```diff
@@ -113,31 +615,19 @@
       // If pageProgression is not set RTL we assume it is LTR
       if (0 == (index & 0x1)) {
         // Even-numbered page
-        if (false === this.hasCover) {
```

Comment:
@patdunlavey @giancarlobi @DigitLib this seems off. We had cover support enabled. @patdunlavey do you remember why this was removed during the rebase?